scrapy/utils/response.py

```diff
@@ -80,8 +81,9 @@ def open_in_browser(
     body = response.body
     if isinstance(response, HtmlResponse):
         if b'<base' not in body:
-            repl = f'<head><base href="{response.url}">'
-            body = body.replace(b'<head>', to_bytes(repl))
+            repl = fr'\1<base href="{response.url}">'
+            body = re.sub(b"<!--.*?-->", b"", body, flags=re.DOTALL)
```

Comment:
I don’t think this is needed, worst case scenario we replace the head in comments as well, which should not be a huge problem.

Reply:
I agree, it is not an issue for the feature itself, but it makes testing quite hard, to ensure there is a single `<base>` tag.